Source/Manager.swift

```diff
@@ -9,6 +9,10 @@
 import SwiftyJSON
 
 public class Manager: NSURLProtocol {
+    typealias queryArtifacts = [String: [String]]?
+    typealias IDArtifacts = [Int]?
```

Comment:
1. Does it make sense to hoist these somewhere where they can be reused in the data function of ResourceProtocol?
1. Should `queryArtifact` and `nameArtifact` be capitalized?